happi/tests/test_client.py

```diff
@@ -299,10 +322,21 @@ def test_find_cfg(happi_cfg: str):
 
 
 def test_from_cfg(happi_cfg: str):
+    # happi_cfg modifies environment variables to make config discoverable
+    client = Client.from_config()
+    # Internal db path should be constructed relative to the happi cfg dir
+    expected_db = os.path.abspath(os.path.dirname(happi_cfg) + '/db.json')
```

Comment:
For completeness we should use `os.path.join` instead of assuming unix-like paths
```suggestion
    expected_db = os.path.abspath(os.path.join(os.path.dirname(happi_cfg), 'db.json'))
```